source/includes/SystemInformation/system_information.md

```diff
@@ -22,7 +22,96 @@
+            "SystemInfo.JavaRuntimeEnv.fileSystemEncoding": "UTF-8",
+            "SystemInfo.JavaRuntimeEnv.userDirectory": "/usr/local/tomcat",
+            "SystemInfo.JavaRuntimeEnv.tempDirectory": "/usr/local/tomcat/temp"
+        },
+        "SystemInfo.title.memoryInformation": {
+            "SystemInfo.Memory.totalMemory": "607 MB",
+            "SystemInfo.Memory.freeMemory": "350 MB",
+            "SystemInfo.Memory.maximumHeapSize": "683 MB"
+        },
+        "SystemInfo.title.dataBaseInformation": {
+            "SystemInfo.Database.name": "\"openmrs-db\"",
+            "SystemInfo.Database.connectionURL": "jdbc:mysql://openmrs-referenceapplication-mysql:3306/\"openmrs-db\"?autoReconnect=true&sessionVariables=default_storage_engine=InnoDB&useUnicode=true&characterEncoding=UTF-8",
+            "SystemInfo.Database.userName": "\"openmrs-user\"",
+            "SystemInfo.Database.driver": null,
+            "SystemInfo.Database.dialect": null
+        },
+        "SystemInfo.title.moduleInformation": {
+            "SystemInfo.Module.repositoryPath": "/usr/local/tomcat/.OpenMRS/modules",
+            "Atlas Module": "2.2 ",
+            "Form Entry App Module": "1.4.2 ",
+            "Reporting": "1.20.0 ",
+            "Metadata Sharing": "1.6.0 ",
+            "ID Generation": "4.5.0 ",
+            "Allergy UI Module": "1.8.2 ",
+            "EMR API Module": "1.28.0 ",
+            "Registration App Module": "1.16.0 ",
+            "HTML Form Entry UI Framework Integration Module": "1.10.0 ",
+            "App Framework Module": "2.14.0 ",
+            "Reporting REST": "1.11.0 ",
+            "Reference Metadata Module": "2.10.2 ",
+            "Metadata Mapping": "1.3.4 ",
+            "Admin UI Module": "1.3.0 ",
+            "OpenMRS UI Framework": "3.17.0 ",
+            "Reference Application Module": "2.10.0 ",
+            "Metadata Deploy": "1.11.0 ",
+            "App UI Module": "1.12.0 ",
+            "Reporting Compatibility": "2.0.6 ",
+            "HTML Widgets": "1.10.0 ",
+            "Serialization Xstream": "0.2.14 ",
+            "Address Hierarchy": "2.11.0 ",
+            "Registration Core Module": "1.9.0 ",
+            "Attachments": "2.2.0 ",
+            "Core Apps Module": "1.28.0 ",
+            "Event Module": "2.7.0 ",
+            "Provider Management Module": "2.10.0 ",
+            "Calculation": "1.2 ",
+            "Appointment Scheduling UI Module": "1.9.0 ",
+            "Open Web Apps Module": "1.10.0 ",
+            "HTML Form Entry": "3.10.0 ",
+            "FHIR Module": "1.20.0 ",
+            "UI Commons Module": "2.12.0 ",
+            "Reporting UI Module": "1.6.0 ",
+            "Appointment Scheduling Module": "1.12.0 ",
+            "Rest Web Services OMOD": "2.28.0.df1459 ",
+            "Legacy UI Module": "1.6.0 ",
+            "Data Exchange Module": "1.3.4 ",
+            "Reference Demo Data Module": "1.4.5 ",
+            "UI Library Module": "2.0.6 "
+        }
+    }
+}
 
```

Comment:
@Akayeshmantha  I would like your view on how should I depict this response it seems quite large ?

Reply:
that's fine let's give it as it is